web/src/tippyjs.ts

```diff
@@ -196,6 +196,20 @@ export function initialize(): void {
         },
     });
 
+    delegate("body", {
+        target: ".subsection-changes-save",
```

Comment:
This selector is not scoped properly; it seems like it would apply to ALL saved buttons.

Reply:
Apologies, I missed this major issue in my review.

Reply:
> A good starting point over here would be to inspect the `enable_or_disable_save_button` method at `web/src/settings_components.js`

Hi @sayamsamal, we have specified the target path so that it only includes the "save button" for "Other settings". Is this the correct approach to only enable the tippy for that specific button?